Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -80,6 +80,12 @@ def customisePostEra_Run2_2018_express_trackingOnly(process):
     customisePostEra_Run2_2017_express_trackingOnly(process)
     return process
 
+def customisePostEra_Run2_2018_pp_on_AA_express_trackingOnly(process):
+    customisePostEra_Run2_2018_pp_on_AA(process)
+    from Calibration.TkAlCaRecoProducers.PCLHPbeamspot_custom import customise_HPbeamspot as _customise_HPbeamspot
+    _customise_HPbeamspot(process)
```

Comment:
can we simply call ```customisePostEra_Run2_2018_express_trackingOnly```?
also, the other pp_on_AA is 10 lines below and it may be more clear to group by era.

Reply:
Hi @slava77 , currently yes, we can simply call customisePostEra_Run2_2018_express_trackingOnly.
My idea is to have customisePostEra_Run2_2018_pp_on_AA explicitly in the function, so that any update in customisePostEra_Run2_2018_pp_on_AA in the future can be deployed in this new scenario.

Sure, I can put it together with the other pp_on_AA.

Reply:
> can we simply call `customisePostEra_Run2_2018_express_trackingOnly`?
Hello Slava,
  

Hi @slava77 , 
My idea is to have customisePostEra_Run2_2018_pp_on_AA explicitly in the function, so that any update in customisePostEra_Run2_2018_pp_on_AA in the future can be reflected in this new scenario.
But sure for now, simply calling customisePostEra_Run2_2018_express_trackingOnly will work.
What do you think?

And sure, I should put it together with the other pp_on_AA era.

Reply:
Fixed in the latest commit.